internal/api/server.go

```diff
@@ -490,7 +490,7 @@ func setPageSecurityHeaders(w http.ResponseWriter, r *http.Request) {
 	// The graphql playground pulls its frontend from a cdn
 	if r.URL.Path == playgroundEndpoint {
 		connectSrc += " https://cdn.jsdelivr.net"
-		scriptSrc += " https://cdn.jsdelivr.net"
+		scriptSrc += " https://cdn.jsdelivr.net http://www.gstatic.com https://www.gstatic.com"
```

Comment:
I haven't actually run this branch, but these lines are only run in the graphql playground. Is this correct?

Reply:
I see I didn't pay close enough attention to that if statement when merging conflicts. This change should actually be moved to line 482. This is the bit that enables the `cast_sender.js` dependency located at  https://www.gstatic.com/cv/js/sender/v1/cast_sender.js?loadCastFramework=1 to load.